pages/ios/guide.js

```diff
@@ -1,11 +1,15 @@
 import React, { Component } from 'react';
 import GuideTemplate from '../../components/GuideTemplate';
 
+import store from '../../store/ios';
+
 export default class extends Component {
   render() {
     return (
       <GuideTemplate
         sectionName="iOS SDK"
+        menu={store.guide.menu}
+        activeMenu="How to install SkyKit SDK in Xcode"
```

Comment:
So it is okay for the `Menu` component to render its activeness according to `window.location` ?